components/ui/ui-DropDownMenu.R

```diff
@@ -183,7 +77,6 @@ actionMenu <- function(..., size = "default", status = "default", icon = NULL, m
         },
         error = function(w) {}
       )
-    ),
-    dropdown.assets(id)
+    )
```

Comment:
Are we sure this isn't also needed for `ActionMenu()`?

Reply:
Good catch. Just fixed it